src/pages/MainPage.jsx

```diff
@@ -6,7 +6,7 @@ import { SearchForm } from "../components/Projects/SearchForm";
 import { ProjectsList } from "../components/Projects/ProjectsList";
 
 export function MainPage() {
-  if (user().login === "admin" && user().pass === "1234") {
+  if (user().isLogged === true) {
```

Comment:
same, please use selectors inside component and feel free to use boolean variables as they are
```suggestion
 const { isLogged } = useSelector(userSelector);
  if (isLogged) {
```